client/src/cordova/apple/OutlineAppleLib/Package.swift

```diff
@@ -47,18 +48,6 @@ let package = Package(
                 "OutlineNotification",
             ]
         ),
-        .target(
```

Comment:
It turns out no other target depends on `Tun2socks`, so after removing the unused references, I was able to move PacketTunnelProvider into the side project and out of the Swift Package!